javascript-primitives-and-variables/main.js

```diff
@@ -0,0 +1,15 @@
+var fullName = 'moamle hashim';
+var isCool = true;
+var totalPets = 1;
+var unknown = null;
+var never =
+console.log('value of full name:', fullName);
+console.log('value of isCool:', isCool);
+console.log('value of totalPets:', totalPets);
+console.log('value of unknown:', unknown);
+console.log('value of never:', never);
+console.log('typeof fullName:', fullName);
```

Comment:
you have to make sure to use the `typeof` operator for the console logs where you want the type of the value